controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -172,23 +169,6 @@ func (r *OCSInitializationReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&ocsv1.OCSInitialization{}).
 		Owns(&appsv1.Deployment{}).
-		Watches(&source.Kind{Type: &configv1.ClusterVersion{}}, handler.EnqueueRequestsFromMapFunc(func(obj client.Object) []reconcile.Request {
```

Comment:
I think we should watch configmap. once the storagecluster is deleted it will also get deleted and rook won't come up after it gets restarted. Basically, we always need this configmap to be present.